src/test/regress/expected/limit_intermediate_size.out

```diff
@@ -133,12 +133,10 @@ WITH cte AS (
 	SELECT * FROM cte2, cte3 WHERE cte2.value_1 IN (SELECT value_2 FROM cte3)
 )
 SELECT count(*) FROM cte;
- count
----------------------------------------------------------------------
-  1824
-(1 row)
-
--- this will fail in real_time_executor
+ERROR:  the intermediate result size exceeds citus.max_intermediate_result_size (currently 3 kB)
```

Comment:
This test has been changed in https://github.com/citusdata/citus/pull/3850/files#r503740858

I guess when we set `task-tracker`, we were not able to use local execution. When we  use adaptive executor, we can. And, in the former case, the size was exceeding, but because of #4235, adaptive executor failed